aws/backoff_retry_lambda/data_lookups.tf

```diff
@@ -1,3 +1,6 @@
+data "aws_region" "current" {}
+data "aws_caller_identity" "current" {}
```

Comment:
We should be able to get this from the `account_id` variable in the common_variables.tf file instead of having to look it up.